src/main/kotlin/io/github/mojira/arisa/infrastructure/config/Arisa.kt

```diff
@@ -99,31 +108,51 @@ object Arisa : ConfigSpec() {
 
         object Empty : ModuleConfigSpec() {
             val emptyMessage by Crash.optional(
-                "We are unable to diagnose your issue due to the lack of proper debug information. " +
-                        "Please review the [guidelines|http://help.mojang.com/customer/portal/articles/801354-writing-helpful-bug-reports-for-minecraft] before reporting issues.\r\n" +
-                        "In case of a game crash, please also attach the crash log from " +
-                        "{{[minecraft/crash-reports/crash-<DATE>-client.txt|http://hopper.minecraft.net/help/guides/finding-minecraft-data-folder/]}}."
-            )
+                "*Thank you for your report!*\n" +
+                        "However, this issue is {color:#FF5722}*Incomplete*{color}.\n" +
+                        "\n" +
+                        "Your report does not contain enough information. As such, we're unable to understand or reproduce the problem.\n" +
+                        "Please review the guidelines linked below before making further reports.\n" +
+                        "\n" +
+                        "In case of a game crash, be sure to attach the crashlog from {{[minecraft/crash-reports/crash-<DATE>-client.txt|https://minecrafthopper.net/help/guides/finding-minecraft-data-folder/]}}.\n" +
+                        "\n" +
+                        "*Quick Links*:\n" +
+                        "\uD83D\uDCD3 [Issue Guidelines|https://bugs.mojang.com/projects/MC/summary] -- \uD83D\uDCAC [Community Support|https://discord.gg/58Sxm23] -- \uD83D\uDCE7 [Customer Support|https://help.minecraft.net/hc/en-us/requests/new] -- \uD83D\uDCD6 [Game Wiki|https://minecraft.gamepedia.com/Minecraft_Wiki]"
+                "{color:#bbb}-- I am a bot. This action was performed automagically! Please report any issues in [Discord|https://discordapp.com/invite/rpCyfKV] or [Reddit|https://www.reddit.com/r/Mojira/]{color}"
+                )
```

Comment:
Still need to do the same thing at other places ;)
The one you did do looks good tho

Reply:
Also probably need to add a tab in front of color, just to make the ktlint check happy at the end.